src/rules/commentFormatRule.ts

```diff
@@ -39,20 +39,25 @@ export class Rule extends Lint.Rules.AbstractRule {
         options: {
             type: "array",
             items: {
-                type: "string",
-                enum: ["check-space", "check-lowercase", "check-uppercase"],
+                oneOf: [{
+                    type: "string",
+                    enum: ["check-space", "check-lowercase", "check-uppercase"],
+                }, {
+                    type: "array",
```

Comment:
I'm not that familiar with JSON schema, but don't you want `anyOf` instead of `oneOf`?